contrib/epee/include/console_handler.h

```diff
@@ -365,21 +357,13 @@ namespace epee
           {
             continue;
           }
-          else if(cmd_handler(command))
-          {
-            continue;
-          }
           else if(0 == command.compare("exit") || 0 == command.compare("q"))
```

Comment:
This line isn't your code of course, but I just noticed this ugliness here: this is a pointlessly obfuscated way of writing `else if (command == "exit" || command == "q")`.